pyproject.toml

```diff
@@ -76,7 +76,7 @@ exclude=[
 # folder.
 "examples/*"=["E402"]
 "doc/conf.py"=["E402"]
-
+"sklearn/metrics/_pairwise_distances_reduction/__init__.py"=["E501"]
```

Comment:
Why is that needed?

Reply:
The diagram's width takes more than 88 characters.

Reply:
Could you instead release the ruff rule for those lines? I think that’s possible.

Reply:
Done in b065c26, practical although slightly annoying. This syntax is not yet supported by `ruff`:

```python
fmt: off

# ...

fmt: on
```